src/app/pages/instances/components/instance-wizard/instance-wizard.component.html

```diff
@@ -143,6 +143,15 @@
       </ix-form-section>
     }
 
+    <ix-form-section [label]="'Storage' | translate">
+      <ix-select
+        formControlName="storage_pool"
+        [label]="'Pool' | translate"
+        [options]="poolOptions$"
+        [hint]="'Select a storage pool for the root filesystem. If left empty, the default pool from the global configuration will be used. This setting cannot be changed after creation.' | translate"
```

Comment:
I would:
- Rename to "Store At"
- Not show when there is only one option available.
- Default to first option.
- Make required.
- Not show hint.